### PR TITLE
optimized RNNBatchStep source code

### PR DESCRIPTION
**Pull request by joyalbin**
TODO(mirkov,raziel): replace this for-loop with a MACRO (or function)

### CODE REVIEW

**Comment by gbaned** [comment]
@aselle Could you PTAL and approve.

File: tensorflow/lite/kernels/internal/kernel_utils.cc [hunk 1]
```diff
@@ -137,6 +137,16 @@ void RnnBatchStep(
                hidden_state_ptr_batch, output_ptr_batch);
 }
 
+#define TFLITE_QUANTIZE_GET_MIN_MAX(units, batch, quantized_batch, scale) \
```
Comment (talumbau) [inline tensorflow/lite/kernels/internal/kernel_utils.cc]: Do you have data that suggests this macro is faster than an inlined function? Overall I think this is good to get rid of code duplication, but would only go this route if we had evidence that an inline function here was slower.

**Review by talumbau — CHANGES_REQUESTED** [review]

**Comment by gbaned** [comment]
@joyalbin Could you please address the reviewer comments. Thanks!

**Comment by mellanox-github** [comment]
Can one of the admins verify this patch?

**Comment by gbaned** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!

**Pull request closed without merging**